src/main/java/com/faforever/client/remote/domain/GameLaunchMessage.java

```diff
@@ -1,50 +1,33 @@
 package com.faforever.client.remote.domain;
 
+import com.faforever.client.fa.relay.LobbyMode;
+import com.faforever.client.game.Faction;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+
 import java.util.List;
 
+@Getter
+@Setter
 public class GameLaunchMessage extends FafServerMessage {
 
+  /**
+   * Stores game launch arguments, like "/ratingcolor d8d8d8d8", "/numgames 236".
+   */
   private List<String> args;
   private int uid;
   private String mod;
   private String mapname;
+  @NonNull
+  private String name;
+  private int expectedPlayers;
```

Comment:
So this would be the number of players when the lobby is full? I don't think the server implements it correctly yet. It'll be easy to fix though.

Reply:
Yes, the autolobby starts the game if the number of expectedPlayers is connected. Until then you are in the connecting stage.